walkthroughs/week-4-tdd/project/calendar-walkthrough.md

```diff
@@ -74,7 +74,7 @@ mvn test
 When all the tests pass, you can be confident that your code works!
 
 **Tip**: You execute a single test method by passing the flag
-`-Dtest=FindMeetingQueryTest#testMethodName`. E.g. To run just the test method 
+`-Dtest=FindMeetingQueryTest#testMethodName`. E.g. To run just the test method
 named `optionsForNoAttendees()` you would execute:
```

Comment:
oops, looks like the linter automatically removed a trailing whitespace. I'll undo this change before merging